ios/BurntModule.swift

```diff
@@ -235,34 +235,34 @@ public class BurntModule: Module {
     }.runOnQueue(.main)
     
     AsyncFunction("alertAsync")  { (options: AlertOptions) -> Void in
-      var preset: SPAlertIconPreset?
+      var preset: AlertIcon?
       do {
-        preset = try options.preset.toSPAlertIconPreset(options)
+        preset = try options.preset.toAlertIcon(options)
       } catch {
         log.error("Burnt Alert error: \(error)")
       }
+
+      guard let window = UIApplication.shared.windows.filter({ $0.isKeyWindow }).first else { return }
       
-      let view = SPAlertView(
+      let view = AlertAppleMusic16View(
         title: options.title,
-        message: options.message,
-        preset: preset ?? .done)
+        subtitle: options.message,
+        icon: preset ?? .done
+      )
       
       if let duration = options.duration {
         view.duration = duration
       }
       
       view.dismissByTap = options.shouldDismissByTap
-      
-      if let icon = options.layout?.iconSize {
```

Comment:
AlertKit no longer exposes the view.layout (see https://github.com/sparrowcode/AlertKit/blob/e8a0208c1af3e0fc1393eb3d58381d517811c0d8/Sources/AlertKit/Views/AlertAppleMusic16View.swift#L182). Should we remove it or update the upstream lib?